packages/components/src/Box.js

```diff
@@ -9,10 +10,19 @@ const shouldForwardProp = createShouldForwardProp([
   ...color.propNames,
 ])
 
-const sx = props => css(props.sx)(props.theme)
-const base = props => css(props.__css)(props.theme)
-const variant = ({ theme, variant, __themeKey = 'variants' }) =>
-  css(get(theme, __themeKey + '.' + variant, get(theme, variant)))
+const sx = (props) => css(props.sx)(props.theme)
+const base = (props) => css(props.__css)(props.theme)
+const variant = ({ theme, variant, __themeKey = 'variants' }) => {
+  if (Array.isArray(variant)) {
+    return css(
+      deepmerge.all(
```

Comment:
shouldn't we also support

```
sx={{
  variant: ['a', 'b']
}}
```

?

Reply:
We could move this logic up to `css` package.

Reply:
> shouldn't we also support
> 
> ```
> sx={{
>   variant: ['a', 'b']
> }}
> ```
> 
> ?

This is the responsive variants syntax, right?

Reply:
> This is the responsive variants syntax, right?

Well, now it is.

_from V1 roadmap https://github.com/system-ui/theme-ui/issues/832_

> - [ ] Variant composition, e.g. variant: ['large', 'primary'] (or plural variants). This would potentially be a breaking change, meaning that the variant property itself would no longer support responsive arrays, but the values within the variant could. #586, #1017

```
sx={{
  variant: ['a', 'b'] // new combined variant
  variant: [['a', 'b']] // new responsive tuple variant
}}
```

Reply:
I think we should create a new issue with that feature.